valio-master/Valio/ItemTableViewCell.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  ItemTableViewCell.swift
+//  Valio
+//
+//  Created by Sam Soffes on 6/6/14.
+//  Copyright (c) 2014 Sam Soffes. All rights reserved.
+//
+
+import UIKit
+import QuartzCore
+
+class ItemTableViewCell: UITableViewCell {
+	
+	// MARK: - Properties
+
+	lazy var titleLabel: UILabel = {
+		let label = UILabel()
```

Comment:
Just a style issue, it's probably alright, but I would use the designated initializer for views, aka `initWithFrame:`